lib/msal-common/src/index.ts

```diff
@@ -51,15 +51,15 @@ export { ICrypto, PkceCodes, DEFAULT_CRYPTO_IMPLEMENTATION } from "./crypto/ICry
 export { SignedHttpRequest } from "./crypto/SignedHttpRequest";
 // Request and Response
 export { BaseAuthRequest } from "./request/BaseAuthRequest";
-export { AuthorizationUrlRequest } from "./request/AuthorizationUrlRequest";
-export { AuthorizationCodeRequest } from "./request/AuthorizationCodeRequest";
-export { RefreshTokenRequest } from "./request/RefreshTokenRequest";
-export { ClientCredentialRequest } from "./request/ClientCredentialRequest";
-export { OnBehalfOfRequest } from "./request/OnBehalfOfRequest";
-export { SilentFlowRequest } from "./request/SilentFlowRequest";
-export { DeviceCodeRequest } from "./request/DeviceCodeRequest";
-export { EndSessionRequest } from "./request/EndSessionRequest";
-export { UsernamePasswordRequest } from "./request/UsernamePasswordRequest";
+export { CommonAuthorizationUrlRequest } from "./request/CommonAuthorizationUrlRequest";
+export { CommonAuthorizationCodeRequest } from "./request/CommonAuthorizationCodeRequest";
+export { CommonRefreshTokenRequest } from "./request/CommonRefreshTokenRequest";
+export { CommonClientCredentialRequest } from "./request/CommonClientCredentialRequest";
+export { CommonOnBehalfOfRequest } from "./request/CommonOnBehalfOfRequest";
+export { CommonSilentFlowRequest } from "./request/CommonSilentFlowRequest";
+export { CommonDeviceCodeRequest } from "./request/CommonDeviceCodeRequest";
+export { CommonEndSessionRequest } from "./request/CommonEndSessionRequest";
+export { CommonUsernamePasswordRequest } from "./request/CommonUsernamePasswordRequest";
```

Comment:
These would all warrant a breaking change in common, fyi.

Reply:
Yes, they would.